vector/src/main/java/im/vector/app/core/utils/PermissionsTools.kt

```diff
@@ -284,6 +289,19 @@ private fun checkPermissions(permissionsToBeGrantedBitMap: Int,
     return isPermissionGranted
 }
 
+fun VectorBaseActivity.onPermissionDeniedSnackbar(@StringRes rationaleMessage: Int) {
+    showSnackbar(getString(rationaleMessage), R.string.settings) {
+        tryOrNull {
+            startActivity(
+                    Intent().apply {
+                        action = Settings.ACTION_APPLICATION_DETAILS_SETTINGS
+                        addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
+                        data = Uri.fromParts("package", this@onPermissionDeniedSnackbar.packageName, null)
+                    })
```

Comment:
This code is already in `VectorSettingsHelpAboutFragment`. Move it to the file `ExternalApplicationsUtil.kt` to avoid code duplication?

Reply:
ok